i18n/en.json

```diff
@@ -1028,10 +1028,6 @@
     "translation": "Email batching job ran. %v user(s) still have notifications pending."
   },
   {
-    "id": "api.email_batching.check_pending_emails.status.app_error",
```

Comment:
Removed this string. It was only use for internationalization of log strings, which is a pain during development. Not certain if the string needs to be removed from other language's files too?